modules/adhashBidAdapter.js

```diff
@@ -0,0 +1,90 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER } from '../src/mediaTypes.js';
+
+export const spec = {
+  code: 'adhash',
+  url: 'https://bidder.adhash.org/rtb?version=1.0&prebid=true',
```

Comment:
Could set the version value explicitly with a variable for readability and incrementing later.

Reply:
Thanks for that suggestion. I changed it as suggested.